test/fixtures/rules/std_rule_cases/L042.yml

```diff
@@ -30,17 +156,38 @@ with_fail:
         )
         select * from d
     ) using (x)
+  fix_str: |
+    with prep_1 as (
+        with d as (
+            select x, z from b
+        )
+        select * from d
+    )
+    select
+        a.x, a.y, b.z
+    from a
+    join prep_1 using (x)
```

Comment:
@barrywhart How should this Fix handle this.
Nested Subqueries are silly. But that is not the scoped name of this Fix.

PS can/should I do a rule fo nested WITH?

Reply:
I think nested `WITH` could be useful in some cases. Not that you technically need it, but for modularity. For example, outer CTE `a` is built using nested CTEs `a1`, `a2`, and `a3`, then outer CTE `b` is built using nested CTEs `b1` and `b2`.